tests/lib/thrift/ThriftTest/src/context.hpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2022 Young Mei
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <memory>
+
+#include <pthread.h>
+
+#include <thrift/server/TSimpleServer.h>
+
+#include "ThriftTest.h"
+
+using namespace apache::thrift::server;
+using namespace thrift::test;
+
+struct ctx {
+  enum {
+    SERVER,
+    CLIENT,
+  };
+
+  std::array<int, CLIENT + 1> fds;
```

Comment:
Different spacing in hpp to cpp files

Reply:
This `context.hpp` file is using 4 space for indents, the `main.cpp` file in the same directory is using tabs, e.g.:

```
static void *server_func(void *arg)
{
	(void)arg;
```

Reply:
I see. Thanks for clarifying. Yeah, it's kind of unfortunate right now, the `.clang-format` file we have only does really horrible things to C++ files. Let me take another look.

Reply:
Fixed!